src/data/markdown/docs/01 guides/02 Using k6/14 Executors.md

```diff
@@ -0,0 +1,295 @@
+| `rate`            | integer | Number of iterations to execute each `timeUnit` period.                                 | -       |
+| `timeUnit`        | string  | Period of time to apply the `rate` value.                                               | `"1s"`  |
+| `duration`        | string  | Maximum test duration before it's forcibly stopped (excluding `gracefulStop`).          | -       |
+| `preAllocatedVUs` | integer | Number of VUs to pre-allocate before test start in order to preserve runtime resources. | -       |
+| `maxVUs`          | integer | Maximum number of VUs to allow during the test run.                                     | -       |
+
+
+### Ramping arrival rate
+
+A variable number of iterations are executed in a specified period of time. This is
+similar to the ramping VUs executor, but for iterations instead, and k6 will attempt
+to dynamically change the number of VUs to achieve the configured iteration rate.
+
+| Option            | Type    | Description                                                                             | Default |
+|-------------------|---------|-----------------------------------------------------------------------------------------|---------|
+| `startRate`       | integer | Number of iterations to execute each `timeUnit` period at test start.                   | `0`     |
+| `timeUnit`        | string  | Period of time to apply the `startRate` the `stages` `target` value.                    | `"1s"`  |
+| `stages`          | array   | Array of objects that specify the target number of iterations to ramp up or down to.    | `[]`    |
+| `preAllocatedVUs` | integer | Number of VUs to pre-allocate before test start in order to preserve runtime resources. | -       |
+| `maxVUs`          | integer | Maximum number of VUs to allow during the test run.                                     | -       |
+
+
+## Shared configuration
+
+All executors share a base configuration, so the following options can be used for all:
+
+| Option         | Type   | Description                                                                      | Default     |
+|----------------|--------|----------------------------------------------------------------------------------|-------------|
+| `startTime`    | string | Time offset since test start this scenario should begin execution.               | `"0s"`      |
+| `gracefulStop` | string | Time to wait for iterations to finish executing before stopping them forcefully. | `"30s"`     |
+| `exec`         | string | Name of exported JS function to execute.                                         | `"default"` |
+| `env`          | object | Environment variables specific to this scenario.                                 | `{}`        |
+| `tags`         | object | [Tags](/using-k6/tags-and-groups) specific to this scenario.                     | `{}`        |
+
```

Comment:
I considered that, but then felt it was better to introduce all executors first, otherwise these options could be confusing.

But you might be right, I'll move it to the top. :)

Reply:
Resolved in f99fbf3.